client/src/sections/material/AddMaterialModal.tsx

```diff
@@ -0,0 +1,67 @@
+import Button from '@/src/shared/components/Button';
+import Modal from '@/src/shared/components/Modal/Modal';
+import TabButton from '@/src/shared/components/TabButton';
+import React, { Fragment, useState } from 'react';
+import SingleFileForm from './SingleFileForm';
+import YouTubeForm from './YoutubeForm';
+import EmbedLinkForm from './EmbedLinkForm';
+
+const AddMaterialModal: React.FC = () => {
+  const [isAddModalOpen, setIsAddModalOpen] = useState(false);
+
+  const handleAddMaterialModal = (): void => {
+    setIsAddModalOpen(!isAddModalOpen);
+  };
+
+  const [activeTab, setActiveTab] = useState('singleFile');
+
+  const handleTabClick = (tab: string): string => {
+    setActiveTab(tab);
+    return tab;
+  };
+
+  return (
+    <Fragment>
+      <Button text="Add Material" onClick={handleAddMaterialModal} />
+
+      <Modal className="w-2/3" isOpen={isAddModalOpen}>
```

Comment:
Please give this modal a fixed height, ideally it should have the same space at the bottom as it is at the top, currently if the content size is big enough, its touching the bottom of the screen which looks pretty bad

![image](https://user-images.githubusercontent.com/110364690/235052500-eb19598d-0e8a-4ebf-a137-cdc68c90c431.png)